addons/website_slides/static/src/scss/website_slides.scss

```diff
@@ -185,7 +185,7 @@ $line-height-truncate: 1.25em;
     a.o_wslides_js_quiz_is_correct {
         color: black;
         input:checked + i.fa-check-circle-o {
-            color: $primary !important;
+            color: #198754 !important;
```

Comment:
@khpa-odoo Use `$success` class instead.

Reply:
Used `$success` variable instead of `$primary` :+1: